src/languages/en-US.ts

```diff
@@ -2036,6 +2043,37 @@ export default class extends Language {
 			extendedHelp: `This command shows a random starred message or the starboard usage and leaderboard for this server.`
 		}),
 
+		/**
+		 * ##################
+		 * SUGGESTIONS COMMANDS
+		 */
+
+		COMMAND_SUGGEST_DESCRIPTION: 'Hey admins! Please do this and that...',
+		COMMAND_SUGGEST_EXTENDED: builder.display('suggest', {
+			extendedHelp: 'Posts a suggestion to the server\'s suggestion channel',
+			explainedUsage: [
+				['suggestion', 'Your suggestion']
+			],
+			examples: [
+				'let\'s make a music channel'
+			],
+			reminder: 'You need to have setup a suggestions channel for this command to work. If you are an administrator, you will be given the chance to do so upon invoking the command'
+		}),
+		COMMAND_RESOLVESUGGESTION_DESCRIPTION: 'Mark a suggestion as accepted, considered, or even denied',
+		COMMAND_RESOLVESUGGESTION_EXTENDED: builder.display('resolvesuggestion', {
+			extendedHelp: `This command marks a suggestion as accepted, considered, or denied.
+			It also can be configured to DM the author regarding the status of their suggestion, with the \`suggestions.on-action.dm\` setting.
+			Also, in case you wish to preserve anonymity, you can hide your name using the \`suggestions.on-action\` setting, which can be overridden with the \`--hide-author\` and \`--show-author\` flags`,
```

Comment:
```suggestion
			Furthermore, in case you wish to preserve anonymity, you can hide your name using the \`suggestions.on-action\` setting, which can be overridden with the \`--hide-author\` and \`--show-author\` flags`,
```